responsibleai/responsibleai/databalanceanalysis/feature_balance_measures.py

```diff
@@ -0,0 +1,419 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import itertools
+from typing import Callable, Dict, List, Tuple
+
+import numpy as np
+import pandas as pd
+import scipy
+
+from responsibleai.databalanceanalysis.balance_measures import BalanceMeasures
+from responsibleai.databalanceanalysis.constants import Constants, Measures
+
+
+class _MeasureHelper:
```

Comment:
same nit here, I think it would be better to just make these methods non-static in the file and remove the class